includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1342,8 +1341,11 @@ public function process_payment_for_order( $cart, $payment_information, $additio
 			} else {
 				$save_user_in_platform_checkout = false;
 
+				$session_data = WC()->session->get( 'platform-checkout-user-data' );
```

Comment:
```suggestion
				$session_data = WC()->session->get( Platform_Checkout_Extension::PLATMORM_CHECKOUT_SESSION_KEY );
```
I'm not sure that you need this at all, due to another comment, but I Can't Believe It's Not Constant®.